src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,31 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+public class CEO {
+    public CEO(String name)
+    {
+        this.name=name;
+    }
+    private String name;
+    private ArrayList<AbstractEmployee> employees_list = new ArrayList();
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public  ArrayList<AbstractEmployee> getEmployees() {
+        return employees_list;
```

Comment:
If you return a list in this way someone could do this:

```
CEO ceo = new CEO(...);

List<AbstractEmployee> employees = ceo.getEmpoyees();

employees.add(new Employee(...)); 
```

This should not be allowed since it can be very error prone and hard to debug.

So it would be better to return `Collections.unmodifiableList(employeeList)` instead.